### PR TITLE
[INMA1731] Add exam June 2019

### PR DESCRIPTION
**Pull request by MartinBraquet**

### CODE REVIEW

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+One investigates the linear estimation of a zero-mean signal $x[k]$ from a signal $y[k]$ containing a colored (non white) noise. More precisely,
+\begin{equation}
+    y[k]=\sum_{l=0}^{L-1}h[l]x[k-l]+\nu[k],
+\end{equation}
+where
+\begin{itemize}
+    \item $h[l]$ is the finite impulse response of a deterministic causal filter,
+    \item the vector process ($x[k]$ $y[k]$)$^T$ is WSS,
+    \item $\nu[k]$ is a zero-mean additive (uncorrelated with $x[k]$) NON-white noise, with autocovariance function $C_\nu[k]$ and power spectrum density $\gamma_\nu(\exp^{j\Omega})$.
```
[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: Same for other exponentials below.
(Sad that GitHub's Markdown doesn't support LaTeX after all those years.

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    \item Compute the power spectral density of the output Y, i.e. $\gamma_Y(z)$.
+    \item Sketch $\gamma_Y(e^{j\Omega})$. In other words, show how the graph of the function $\Omega \mapsto \gamma_Y(e^{j\Omega})$ looks like. Give an approximation of its local extrema (position and value).
+    \item Realize the system by a difference equation.
+    \item What kind of finite-dimensional model do you obtain ?
+\end{enumerate}
+
+\nosolution
+
+\section{}
+Let Z be an observed random vector in $\mathbb{R}^n$ and $g:\mathbb{R}^n\to \mathbb{R}$ an estimation rule for a stochastic estimand $\Theta$ (Bayesian estimation). Suppose that $\mathbb{E}[g(Z)]=\mathbb{E}[\Theta]$ and that we have a function $h:\mathbb{R}^n \to \mathbb{R}$ such that $\mathbb{E}[h(Z)]=m_{h(Z)}$ is known. Consider the new estimation rule \begin{equation}
```
Comment (Jimvy) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
Let $Z$ be an observed random vector in $\mathbb{R}^n$ and $g \colon \mathbb{R}^n\to \mathbb{R}$ an estimation rule for a stochastic estimand $\Theta$ (Bayesian estimation). Suppose that $\mathbb{E}[g(Z)]=\mathbb{E}[\Theta]$ and that we have a function $h \colon \mathbb{R}^n \to \mathbb{R}$ such that $\mathbb{E}[h(Z)]=m_{h(Z)}$ is known. Consider the new estimation rule \begin{equation}
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+\section{}
+
+Consider the discrete-time LTI causal system with transfer function
+
+\begin{equation}
+    G(z)=\frac{z}{\left(z-\frac{1-\epsilon}{\sqrt{2}}(1+j)\right)\left(z-\frac{1-\epsilon}{\sqrt{2}}(1-j)\right)}
+\end{equation}
+\indent where $0<\epsilon\ll1$. Let the input X be white noise with variance $\sigma_X^2$.
+\begin{enumerate}
+    \item Compute the power spectral density of the output Y, i.e. $\gamma_Y(z)$.
```
Comment (Jimvy) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
    \item Compute the power spectral density of the output $Y$, i.e. $\gamma_Y(z)$.
```
Poor forgotten little dollars :'(

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 4]
```diff
@@ -0,0 +1,64 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\hypertitle{Stochastic processes}{6}{INMA}{1731}{2019}{June}{Majeure}
+{Alice Borbáth}
+{Luc Vandendorpe}
```
Comment (Jimvy) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: Pierre-Antoine Absil is missing

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+\end{enumerate}
+
+We then investigate a solution where the first step is to whiten the noise (make it white). we know that there exists a filter $L_\nu(\exp^{j\Omega})$ such that $\gamma_\nu(\exp^{j\Omega})=L_\nu(\exp^{j\Omega})$. By passing the signal $y[k]$ in a filter with frequency response $\frac{1}{L_\nu(\exp^{j\Omega})}$, we obtain a new signal $y_2[k]$ from which we investigate the MMSE estimation of $x[k]$ with the help of an optimum smoother with coefficients $w_2[k]$, that is 
+\begin{equation}
+    \hat{x}'[k]=\sum_{l=-\infty}^{\infty}w_2[l]\: y_2[k-l]
+\end{equation}
+
+\begin{enumerate}[resume]
+    \item Derive and provide the optimum smoother coefficients $W_2(\exp^{j\Omega})$ for an estimator of $x[k]$ optimized according to the MMSE or Wiener criterion. Express your result as un function of the power spectral densities of $x[k]$ and $\nu[k]$, and $H(\exp^{j\Omega})$.
+    \item How does the estimate $\hat{x}'[k]$ computed by means of $w_2[l]$ compare with $\hat{x}[k]$ obtained thanks to $w[l]$ ? Justify.
```
[the remark this answers is not in the document]
Comment (MartinBraquet) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
    \item How does the estimate $\hat{x}'[k]$ computed by means of $w_2[l]$ compare with $\hat{x}[k]$ obtained thanks to $w[l]$? Justify.
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 4 above)
Comment (MartinBraquet) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
{Pierre-Antoine Absil and Luc Vandendorpe}
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (MartinBraquet) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: Are you sure that it understands the \e?

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 5 above)
[the remark this answers is not in the document]
Comment (MartinBraquet) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
    \item How does the estimate $\hat{x}'[k]$ computed by means of $w_2[l]$ compare with $\hat{x}[k]$ obtained thanks to $w[l]$? Justify.
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: If (it == 'LaTeX') {
    We defined it in `eplcommon.sty`.
} else if (it == 'GitHub') {
    Well, at least the first one should work then. So I suppose it's just not implemented. The Menlo Park company is ahead.
} else {
    throw new IncomprehensionFacialExpression("What do you mean by it?");
}

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 5 above)
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: FWIW, Babel in English handles this automatically I think (or is it the other way around, where it adds the space automatically in French?)

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: ISO standards prescribe constants should be in upright mode in math. I went ahead and followed the convention...

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 2 above)
Comment (Peiffap) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: You're making me feel a certain type of way with those `\colon`s...

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+\begin{equation}
+    G(z)=\frac{z}{\left(z-\frac{1-\epsilon}{\sqrt{2}}(1+j)\right)\left(z-\frac{1-\epsilon}{\sqrt{2}}(1-j)\right)}
+\end{equation}
+\indent where $0<\epsilon\ll1$. Let the input X be white noise with variance $\sigma_X^2$.
+\begin{enumerate}
+    \item Compute the power spectral density of the output $Y$, i.e. $\gamma_Y(z)$.
+    \item Sketch $\gamma_Y(e^{j\Omega})$. In other words, show how the graph of the function $\Omega \mapsto \gamma_Y(e^{j\Omega})$ looks like. Give an approximation of its local extrema (position and value).
+    \item Realize the system by a difference equation.
+    \item What kind of finite-dimensional model do you obtain ?
```
Comment (Peiffap) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: A surplus space in front of the `?` :)

**Comment by Peiffap** [comment]
If all goes well, I should be able to provide solutions for this somewhere in the next two weeks.

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 5 above)
[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: It's the other way around: It adds spaces before punctuation marks if they're not already there, but it doesn't remove them if they're useless.

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (MartinBraquet) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]: Okok if it is in eplcommon.sty.

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex (hunk 6 above)
Comment (MartinBraquet) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
    \item What kind of finite-dimensional model do you obtain?
```

**Comment by MartinBraquet** [comment]
> If all goes well, I should be able to provide solutions for this somewhere in the next two weeks.

NIce !

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    \item Derive and provide the optimum smoother coefficients $W(\exp^{j\Omega})$ for an estimator optimized according to the MMSE or Wiener criterion. Express your result as a function of the power spectral densities of $x[k]$ and $\nu[k]$, and $H(\exp^{j\Omega})$.
+\end{enumerate}
+
+We then investigate a solution where the first step is to whiten the noise (make it white). we know that there exists a filter $L_\nu(\exp^{j\Omega})$ such that $\gamma_\nu(\exp^{j\Omega})=L_\nu(\exp^{j\Omega})$. By passing the signal $y[k]$ in a filter with frequency response $\frac{1}{L_\nu(\exp^{j\Omega})}$, we obtain a new signal $y_2[k]$ from which we investigate the MMSE estimation of $x[k]$ with the help of an optimum smoother with coefficients $w_2[k]$, that is 
+\begin{equation}
+    \hat{x}'[k]=\sum_{l=-\infty}^{\infty}w_2[l]\: y_2[k-l]
+\end{equation}
+
+\begin{enumerate}[resume]
+    \item Derive and provide the optimum smoother coefficients $W_2(\exp^{j\Omega})$ for an estimator of $x[k]$ optimized according to the MMSE or Wiener criterion. Express your result as un function of the power spectral densities of $x[k]$ and $\nu[k]$, and $H(\exp^{j\Omega})$.
```
Comment (Jimvy) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
    \item Derive and provide the optimum smoother coefficients $W_2(\e^{j\Omega})$ for an estimator of $x[k]$ optimized according to the MMSE or Wiener criterion. Express your result as un function of the power spectral densities of $x[k]$ and $\nu[k]$, and $H(\e^{j\Omega})$.
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+\end{itemize}
+One investigates the optimum smoother corresponding to 
+\begin{equation}
+    \hat{x}[k]=\sum_{l=-\infty}^{\infty}w[l]y[k-l].
+\end{equation}
+\begin{enumerate}
+    \item Derive and provide the optimum smoother coefficients $W(\exp^{j\Omega})$ for an estimator optimized according to the MMSE or Wiener criterion. Express your result as a function of the power spectral densities of $x[k]$ and $\nu[k]$, and $H(\exp^{j\Omega})$.
+\end{enumerate}
+
+We then investigate a solution where the first step is to whiten the noise (make it white). we know that there exists a filter $L_\nu(\exp^{j\Omega})$ such that $\gamma_\nu(\exp^{j\Omega})=L_\nu(\exp^{j\Omega})$. By passing the signal $y[k]$ in a filter with frequency response $\frac{1}{L_\nu(\exp^{j\Omega})}$, we obtain a new signal $y_2[k]$ from which we investigate the MMSE estimation of $x[k]$ with the help of an optimum smoother with coefficients $w_2[k]$, that is
```
Comment (Jimvy) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
We then investigate a solution where the first step is to whiten the noise (make it white). we know that there exists a filter $L_\nu(\e^{j\Omega})$ such that $\gamma_\nu(\e^{j\Omega})=L_\nu(\e^{j\Omega})$. By passing the signal $y[k]$ in a filter with frequency response $\frac{1}{L_\nu(\e^{j\Omega})}$, we obtain a new signal $y_2[k]$ from which we investigate the MMSE estimation of $x[k]$ with the help of an optimum smoother with coefficients $w_2[k]$, that is 
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+    \item $h[l]$ is the finite impulse response of a deterministic causal filter,
+    \item the vector process ($x[k]$ $y[k]$)$^T$ is WSS,
+    \item $\nu[k]$ is a zero-mean additive (uncorrelated with $x[k]$) NON-white noise, with autocovariance function $C_\nu[k]$ and power spectrum density $\gamma_\nu(\e^{j\Omega})$.
+\end{itemize}
+One investigates the optimum smoother corresponding to 
+\begin{equation}
+    \hat{x}[k]=\sum_{l=-\infty}^{\infty}w[l]y[k-l].
+\end{equation}
+\begin{enumerate}
+    \item Derive and provide the optimum smoother coefficients $W(\exp^{j\Omega})$ for an estimator optimized according to the MMSE or Wiener criterion. Express your result as a function of the power spectral densities of $x[k]$ and $\nu[k]$, and $H(\exp^{j\Omega})$.
```
Comment (Jimvy) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
    \item Derive and provide the optimum smoother coefficients $W(\e^{j\Omega})$ for an estimator optimized according to the MMSE or Wiener criterion. Express your result as a function of the power spectral densities of $x[k]$ and $\nu[k]$, and $H(\e^{j\Omega})$.
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+\hypertitle{Stochastic processes}{6}{INMA}{1731}{2019}{June}{Majeure}
+{Alice Borbáth \and Olivier Leblanc \and Julien Verecken}
+{Pierre-Antoine Absil and Luc Vandendorpe}
+
+Exam during 3h with 3 questions: 2 questions by PAA et 1 question by LV.
+
+\section{PAA 1}
+
+Let $G(z)$ be an LTI system which is stable and causal. 
+One inputs a white signal $X$ of variance $\sigma_x^2$ in the system and outputs $Y$.
```
Comment (Peiffap) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
One inputs a white noise signal $X$ of variance $\sigma_X^2$ in the system and outputs $Y$.
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+The transfer function is given by :
+\begin{equation}
+    G(z)=\frac{z}{\left(z-\frac{1-\epsilon}{\sqrt{2}}(1+j)\right)\left(z-\frac{1-\epsilon}{\sqrt{2}}(1-j)\right)}
+\end{equation}
+\indent where $0<\epsilon\ll1$. Let the input X be white noise with variance $\sigma_X^2$.
```
Comment (Peiffap) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
where $0<\epsilon\ll1$.
```

This is already written 3 lines higher (and with a different variance, so I picked the one I thought looked the most logical and in tune with the rest of the notations). The indent just looks ugly, but you can keep it if you wish.

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+\begin{enumerate}
+    \item Compute the power spectral density of the output $Y$, i.e. $\gamma_Y(z)$.
+    \item Sketch $\gamma_Y(e^{j\Omega})$. In other words, show how the graph of the function $\Omega \mapsto \gamma_Y(e^{j\Omega})$ looks like. Give an approximation of its local extrema (position and value).
```
Comment (Peiffap) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
    \item Sketch $\gamma_Y(e^{j\Omega})$. In other words, show what the graph of the function $\Omega \mapsto \gamma_Y(e^{j\Omega})$ looks like. Give an approximation of its local extrema (position and value).
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+    \item Realize the system by a recurrence equation.
```
Comment (Peiffap) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
    \item Define the system by a recurrence equation.
```

I'm not too sure about the wording (and this isn't the exam I took so I can't fall back to what the prof wrote) but "realise" seems like an odd way to word it.

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+    \item What kind of finite-dimensional model do you obtain?
+\end{enumerate}
+
+\nosolution
+
+\section{PAA 2}
+
+\section{}
+Let $Z$ be an observed random vector in $\mathbb{R}^n$ and $g \colon \mathbb{R}^n\to \mathbb{R}$ an estimation rule for a stochastic estimation $\Theta$ (Bayesian estimation). Suppose that $\mathbb{E}[g(Z)]=\mathbb{E}[\Theta]$ and that we have a function $h \colon \mathbb{R}^n \to \mathbb{R}$ such that $\mathbb{E}[h(Z)]=m_{h(Z)}$ is known. Consider the new estimation rule
```
Comment (Peiffap) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
Let $Z$ be an observed random vector in $\Rn$ and $g \colon \Rn \to \R$ an estimation rule for a stochastic estimation $\Theta$ (Bayesian estimation). Suppose that $\mathbb{E}[g(Z)]=\mathbb{E}[\Theta]$ and that we have a function $h \colon \Rn \to \R$ such that $\mathbb{E}[h(Z)]=m_{h(Z)}$ is known. Consider the new estimation rule
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+\nosolution
+
+\section{PAA 2}
+
+\section{}
+Let $Z$ be an observed random vector in $\mathbb{R}^n$ and $g \colon \mathbb{R}^n\to \mathbb{R}$ an estimation rule for a stochastic estimation $\Theta$ (Bayesian estimation). Suppose that $\mathbb{E}[g(Z)]=\mathbb{E}[\Theta]$ and that we have a function $h \colon \mathbb{R}^n \to \mathbb{R}$ such that $\mathbb{E}[h(Z)]=m_{h(Z)}$ is known. Consider the new estimation rule 
+\begin{equation}
+    \Tilde{g}(Z)=g(Z)-\alpha(h(Z)-m_{h(Z)}),
+\end{equation}
+where $\alpha \in \mathbb{R}$. Propose a principled way to choose $\alpha$ assuming that the covariance of $[\Theta, g(Z), h(Z)]$ is known.
```
Comment (Peiffap) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
where $\alpha \in \R$. Propose a principled way to choose $\alpha$ assuming that the covariance of $[\Theta, g(Z), h(Z)]$ is known.
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+
+\nosolution
+
+\section{LV 1}
+
+One investigates the linear estimation of a zero-mean signal $x[k]$ from a signal $y[k]$ containing a colored (non white) noise. More precisely,
```
Comment (Peiffap) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
One investigates the linear estimation of a zero-mean signal $x[k]$ from a signal $y[k]$ containing a colored (non-white) noise. More precisely,
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 17]
```diff
@@ -0,0 +1,70 @@
...
+\end{itemize}
+One investigates the optimum smoother corresponding to 
+\begin{equation}
+    \hat{x}[k]=\sum_{l=-\infty}^{\infty}w[l]y[k-l].
+\end{equation}
+\begin{enumerate}
+    \item Derive and provide the optimum smoother coefficients $W(\e^{j\Omega})$ for an estimator optimized according to the MMSE or Wiener criterion. Express your result as a function of the power spectral densities of $x[k]$ and $\nu[k]$, and $H(\e^{j\Omega})$.
+\end{enumerate}
+
+We then investigate a solution where the first step is to whiten the noise (make it white). we know that there exists a filter $L_\nu(\e^{j\Omega})$ such that $\gamma_\nu(\e^{j\Omega})=L_\nu(\e^{j\Omega})$. By passing the signal $y[k]$ in a filter with frequency response $\frac{1}{L_\nu(\e^{j\Omega})}$, we obtain a new signal $y_2[k]$ from which we investigate the MMSE estimation of $x[k]$ with the help of an optimum smoother with coefficients $w_2[k]$, that is
```
Comment (Peiffap) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
We then investigate a solution where the first step is to whiten the noise (make it white). We know that there exists a filter $L_\nu(\e^{j\Omega})$ such that $\gamma_\nu(\e^{j\Omega})=L_\nu(\e^{j\Omega})$. By passing the signal $y[k]$ in a filter with frequency response $\frac{1}{L_\nu(\e^{j\Omega})}$, we obtain a new signal $y_2[k]$ from which we investigate the MMSE estimation of $x[k]$ with the help of an optimum smoother with coefficients $w_2[k]$, that is 
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/Makefile [hunk 18]
```diff
@@ -0,0 +1,3 @@
+SOL=both
```
Comment (Jimvy) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/Makefile]: On n'avait pas dit qu'on éradiquait les SOL= ?

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+{Alice Borbáth \and Olivier Leblanc \and Julien Verecken}
+{Pierre-Antoine Absil and Luc Vandendorpe}
+
+Exam during 3h with 3 questions: 2 questions by PAA et 1 question by LV.
+
+\section{PAA 1}
+
+Let $G(z)$ be an LTI system which is stable and causal. 
+One inputs a white noise signal $X$ of variance $\sigma_X^2$ in the system and outputs $Y$.
+The transfer function is given by :
```
Comment (Jimvy) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
The transfer function is given by:
```
😁

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex [hunk 20]
```diff
@@ -0,0 +1,70 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\hypertitle{Stochastic processes}{6}{INMA}{1731}{2019}{June}{Majeure}
+{Alice Borbáth \and Olivier Leblanc \and Julien Verecken}
+{Pierre-Antoine Absil and Luc Vandendorpe}
+
+Exam during 3h with 3 questions: 2 questions by PAA et 1 question by LV.
```
Comment (Jimvy) [inline src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex]:
```suggestion
Exam during 3h with 3 questions: 2 questions by PAA and 1 question by LV.
```

File: src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/Makefile (hunk 18 above)
Comment (MartinBraquet) [reply src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/Makefile]: Si
```suggestion
```

**Pull request merged**